functions/go/search-replace/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+image: gcr.io/kpt-fn/search-replace
+description: Search and optionally replace fields across all resources.
+categories:
+  - mutator
+imageTags:
+  - unstable
+docPath: README.md
```

Comment:
is there a need for docPath? Either default it to README.md or remove it?

Reply:
I guess we can default it to `README.md` if unspecified.